ko/Ducky.PackageFromNuget/README_ko.md

```
@@ -0,0 +1,236 @@
+이 디렉토리(['Ducky.PackageFromNuget/'](Ducky.PackageFromNuget/))는 모드의 기능을 확장하는 방법을 보여줍니다.：
+
+- ✅ 타사 NuGet 패키지 소개
+- ✅ 외부 라이브러리를 사용하여 데이터 구문 분석(YAML)
+- ✅ 모드 리소스 파일에 액세스
+- ✅ 종속성 패키징 자동 처리
+
+**예제 시나리오：** YamlDotNet 라이브러리를 사용하여 Mod Resources 폴더의 YAML 구성 파일을 읽고 구문 분석합니다.
+
+## 2. 프로젝트 구조
+
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)